### PR TITLE
Remove unnecessary redirects

### PR DESCRIPTION
**Pull request by rcantin-w**
## What does this change?

Related to #11300 & [#11280](https://github.com/wellcomecollection/wellcomecollection.org/issues/11280)

The first commit was just an alphabetical reorder for readability so I'd suggest reviewing this commit by commit for clarity. I aimed to have clear commit messages for all.

I've modified a lot of UIDs in Prismic to then be able to delete a lot of redirects (matched their vanity URL in Prismic).

I've then moved them all to target UID based URLs.

This removes about half the redirections 🎉 

## How to test

You can go confirm my theories about each of them, but we can't actually test this until `Terraform apply` gets run, which I'll do once this is in staging. We can then test them all on there and when happy, apply the changes to production as well.

## How can we measure success?

A clearer, more useful list of redirections, made minimal and readable!

## Have we considered potential risks?

As we can test on staging before, I think this is low risk.

### CODE REVIEW

File: cache/edge_lambdas/src/redirects.ts [hunk 1]
```diff
@@ -19,193 +19,103 @@
...
  */
 export const literalRedirects: Record<string, string> = {
-  '/event-series/Wo1YeCoAACoAZFoN': '/events/Wqkd1yUAAB8sW4By',
-  '/explore': '/stories',
+  '/access': '/pages/accessibility',
+  '/alice-anderson':
+    '/exhibitions/alice-anderson--memory-movement-memory-objects',
+  '/articles/aids-posters-0': '/articles/aids-posters',
+  '/articles/hysteria': '/articles/what-is-hysteria', // "hysteria" already taken by ZN-ELxEAACMABO5a in former UID.
+  '/articles/lustmord': '/articles/WqfvayUAAKsrVreh', // lustmord-and-the-three-perspectives-of-murder -> not published?
```
Comment (rcantin-w) [inline cache/edge_lambdas/src/redirects.ts:27]: [Waiting to hear back from Editorial in Slack](https://wellcome.slack.com/archives/C3N7J05TK/p1729602171817339)

File: cache/edge_lambdas/src/redirects.ts [hunk 2]
```diff
@@ -102,8 +101,7 @@ export const literalRedirects: Record<string, string> = {
   '/whats-on/exhibitions/brains': '/exhibitions/W31EzykAACkAPuXG',
   '/whats-on/exhibitions/infinitas-gracias': '/exhibitions/W31D3CkAACIAPuGN',
   '/whats-on/exhibitions/superhuman': '/exhibitions/W31FpSkAACkAPumK',
-  '/young-people': '/pages/Wuw2MSIAACtd3Ssg',
```
Comment (davidpmccormick) [inline cache/edge_lambdas/src/redirects.ts]: is this removing `/young-people` altogether?

Comment (rcantin-w) [reply cache/edge_lambdas/src/redirects.ts]: Oh yeah you're right, it should be `/young-people`: `pages/young-people`, thank you!

Comment (rcantin-w) [reply cache/edge_lambdas/src/redirects.ts]: Addressed!

File: cache/edge_lambdas/src/redirects.ts (hunk 1 above)
Comment (rcantin-w) [reply cache/edge_lambdas/src/redirects.ts]: While waiting to hear back, I've changed it to point to the UID, simplifies it.

**Pull request merged**